src/pages/faq.astro

```diff
@@ -0,0 +1,9 @@
+---
+import Layout from '../layouts/Default.astro'
+import FAQ from '../components/Faq/Faq.vue'
+
+---
+
+<Layout metaTitle="FAQ | Dev Protocol">
+<FAQ client:visible></FAQ>
+</Layout>
```

Comment:
Missing an endline.
```suggestion
</Layout>

```

Reply:
Will add that.
Also could you tell me why this endline is important?

Reply:
Some IDEs have configured to have that in all files, and not adding that could break those IDEs. 🙂

Reply:
oohh ok Thanks will keep this in mind.
Also can't we automate this? That is on merging PR all files get an endline if it is not already present? 🤔

Reply:
I think we have editorconfig file and it take care of all these things, you have to install the editorconfig plugin acc to IDE, or code editor you use 🤔